### ISSUE TITLE
Reposense report in tP Dashboard missing commit history

### ISSUE THREAD

**Issue by PotatoCombat** [nus-cs2103-AY1920S2/forum#76]
My team has just noticed that Reposense does not seem to show any of our commits since the start of the tP. Although we have changed a significant number of lines in the `master` branch of our team's repo, Reposense seems to have only detected 2 of my commits (which were committed on different days), and none of my team members'. Also, it seems that other teams may be experiencing the same issue too (parts of their GitHub history aren't showing in RepoSense).

I was able to get a Reposense report by running Reposense.jar on my computer, and found that each of us have committed using multiple author names. At first I suspected that multiple author names may have been causing the issue. However, our alternative names are registered in the tP Dashboard config file, so it seems unlikely to be the cause.

Since we are still in the early stages of the tP, we are anxious to solve this problem ASAP. Did we miss a step during the setup of our repo? Or could there be an underlying bug?

**Comment by okkhoy** [comment]
Just to double confirm: "our alternative names are registered in the tP Dashboard config file" => so, G*******eo;PotatoCombat;T**F****C****P*****97 are all the names with which you have committed?

First, don't panic. This is fixable! and it doesn't hurt you in anything in the project. I won't be grading you each week by picking up the commits. I will only look at the trend (and in any case, grading happens only in the end).

Let me check if any of the configurations I have set is messing up the report, or I will bug @Tejas2805  to take a look 😜 He is our resident reposense expert 😄

**Comment by PotatoCombat** [comment]
Yes! I've made commits to our tP under the author names "G*eo" and "PotatoCombat". I've also checked that I used all 3 author names in my iP.

Thanks for the quick reply! 😄

**Comment by Tejas2805** [comment]
@PotatoCombat Nothing to worry. Once it's fixed, all commits will be recorded automatically - old or new. I will take a look into it and get back to you. Not to worry.

**Comment by Tejas2805** [comment]
Add your repo link here

**Comment by PotatoCombat** [comment]
https://github.com/AY1920S2-CS2103T-W17-4/main
Thank you!

**Comment by raniceyue** [comment]
Hello, my team and I are facing the same issue. We have made multiple merges into our main team repo and none of our progress is showing on RepoSense. 

Our team repo is https://github.com/AY1920S2-CS2103T-F09-4/main

**Comment by nathanaelseen** [comment]
Alternatively, you could also use set up [Nelify](https://nus-cs2103-ay1920s2.github.io/addressbook-level3/UsingNetlify.html), and then set-up a local [reposense](https://github.com/reposense/RepoSense/blob/145a1b4dda02ac9fdc42554835e7be98a534b0a0/docs/UsingNetlifyGuide.md), to view the latest and most updated code contributions of your team members!

(In addition, you could also configure Reposense to generate the most updated report after every merged PR!)

**Comment by okkhoy** [comment]
@nathanaelseen local reposense is good only for verification. in the end, the code evaluation happens based on what is see on the dashboard.

**Comment by nathanaelseen** [comment]
Oh I see, we set up a local reposense because we realised that our code contributions at the [dashboard](https://nus-cs2103-ay1920s2.github.io/tp-dashboard) is not quite updated.

Our team repo is; https://github.com/AY1920S2-CS2103T-F09-3/main
